build.gradle

```diff
@@ -91,5 +91,5 @@ subprojects {
 }
 
 ext {
-    fluxCVersion = '0b1dde6c44833f940c8700b29e794c6534617b93'
+    fluxCVersion = '12e475869bab00d0fc7f4357614cab0e3b36a82f'
```

Comment:
This hash includes many changes on `develop` of FluxC that are not in 11.5 so we can't use it as is.